sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1011,6 +1011,8 @@ static int get_slave_info(const struct snd_soc_acpi_link_adr *adr_link,
 	return 0;
 }
 
+static const char * const type_strings[] = {"SimpleJack", "SmartAmp", "SmartMic"};
```

Comment:
commit message: did you mean "This patch suggests to append codec type if there are two or more different types of devices on the same sdw bus"?

We can't change the RVP or Chromebook case with two identical amps on the same link, can we?

Reply:
> commit message: did you mean "This patch suggests to append codec type if there are two or more different types of devices on the same sdw bus"?
> 

Yes

> We can't change the RVP or Chromebook case with two identical amps on the same link, can we?

No, we can't.